apps/nextjs-website/src/components/pageSection/pageSection.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Hero, { SectionDataToHeroProps } from '../Hero';
+import { PageSection } from '@/lib/fetch/types/PageSection';
+
+export function rendering(componentData: PageSection) {
+  // Temporarily disable no-small-switch rule (new cases will be added with PRs for other components)
+  // eslint-disable-next-line no-underscore-dangle,sonarjs/no-small-switch
+  switch (componentData.__component) {
+    case 'sections.hero':
+      const HeroSectionProps = SectionDataToHeroProps(componentData);
+      return (
+        <Hero
+          {...HeroSectionProps}
+          sectionID={componentData.sectionID ?? undefined}
+        />
+      );
+
+    default:
+      return;
+  }
+}
```

Comment:
I would use uppercase for React components, so `src/components/pageSection/pageSection.tsx` becomes `src/components/PageSection/PageSection.tsx`.

```suggestion
import React from 'react';
import Hero from '../Hero';
import { PageSection as PageSectionData } from '@/lib/fetch/types/PageSection';

const PageSection = (props: PageSectionData) => {
  // Temporarily disable no-small-switch rule (new cases will be added with PRs for other components)
  // eslint-disable-next-line no-underscore-dangle,sonarjs/no-small-switch
  switch (props.__component) {
    case 'sections.hero':
      return <Hero {...props} />;
    default:
      return null;
  }
};

export default PageSection;
```